worker/restore.go

```diff
@@ -35,7 +36,7 @@ import (
 )
 
 // RunRestore calls badger.Load and tries to load data into a new DB.
-func RunRestore(pdir, location, backupId string, key x.SensitiveByteSlice) LoadResult {
+func RunRestore(pdir, location, backupId string, key x.SensitiveByteSlice, ctype options.CompressionType, clevel int) LoadResult {
```

Comment:
Seems like too many arguments. Should this function be changed to take in the entire `opt` struct instead?